locales/fr/translation.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
Also what is the difference between template.json and translation.json?

Reply:
translation.json will be used in the app.
i18n-scanner extracts keys from i18n.t(), t(), <Trans> and <Interpolation> -> i18n-scanner searches in template.json for existing keys and translations -> translation.json is generated based on template.json and found keys will be appended. Non existing keys are automatically set to english translation. I can remove this or integrate it in a seperate folder.

Reply:
Yeah this is a bit confusing. Does it mean that we currently have duplicates? I think one .json file per language should be sufficient right?